theme/templates/pages/features.html

```diff
@@ -146,8 +146,8 @@ <h6 class="docs-header">{% trans %}Search &amp; Sort{% endtrans %}</h6>
       </div>
       <div class="feature-indent">
         <img class="responsive drop-shadow" src="{{ SITEURL }}/theme/images/2.0/feature-library.png">
-        <h6 class="docs-header">{% trans %}iTunes and Traktor Library Integration{% endtrans %}</h6>
-        <p>{% trans %}Drop fresh tracks from your iTunes or Traktor library right into your mix.{% endtrans %}</p>
+        <h6 class="docs-header">{% trans %}Universal DJ Library Integration{% endtrans %}</h6>
```

Comment:
```suggestion
        <h6 class="docs-header">{% trans %}External DJ Library Integration{% endtrans %}</h6>
```
because Mixxx doesn't support each and every library format out there.